[2024-07-16 02:46:42,410] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-10-24T00:00:00+00:00 [queued]>
[2024-07-16 02:46:42,445] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-10-24T00:00:00+00:00 [queued]>
[2024-07-16 02:46:42,446] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:42,447] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-16 02:46:42,447] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:42,472] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-10-24T00:00:00+00:00
[2024-07-16 02:46:42,478] {standard_task_runner.py:52} INFO - Started process 3608 to run task
[2024-07-16 02:46:42,493] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2021-10-24T00:00:00+00:00', '--job-id', '998', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpezjj5o7x', '--error-file', '/tmp/tmpxp42kl2z']
[2024-07-16 02:46:42,499] {standard_task_runner.py:77} INFO - Job 998: Subtask second_task
[2024-07-16 02:46:42,602] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2021-10-24T00:00:00+00:00 [running]> on host f3060829e7b4
[2024-07-16 02:46:42,707] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-24T00:00:00+00:00
[2024-07-16 02:46:42,710] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-16 02:46:42,712] {bash.py:158} INFO - Running command: echo hello world, this is the second task, it will run after first task
[2024-07-16 02:46:42,742] {bash.py:169} INFO - Output:
[2024-07-16 02:46:42,743] {bash.py:173} INFO - hello world, this is the second task, it will run after first task
[2024-07-16 02:46:42,745] {bash.py:177} INFO - Command exited with return code 0
[2024-07-16 02:46:42,896] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20211024T000000, start_date=20240716T024642, end_date=20240716T024642
[2024-07-16 02:46:43,101] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-16 02:46:43,137] {local_task_job.py:146} INFO - Task exited with return code 0
